src/views/Statistics.vue

```diff
@@ -2,8 +2,15 @@
      Use of this source code is governed by a BSD-style license that can be
      found in the LICENSE file. -->
 
+// TODO: Style this, and add tabs for User & Team.
 <template>
-  <StatisticsList v-if="loaded" :items="items" />
+  <div v-if="haveStats">
+    <h1 class="pl-2 py-2">You</h1>
+    <StatisticsList :items="itemsUser" />
+    <hr />
+    <h1 class="pl-2 py-2" v-if="itemsTeam.length">Team</h1>
```

Comment:
it looks like the second `StatisticsList` is still present when there's no team, since the `v-if` only appears here. try using `<template>` to wrap multiple items:

```html
<template v-if="haveStats">
  <h1 ...>You</h1>
  <StatisticsList ... />
  <template v-if="itemTeams.length">
    <hr />
    <h1 ...>Team</h1>
    <StatisticsList ... />
  </template>
</template>
<Spinner v-else />
```

Reply:
I wrapped the second StatisticsList, but I need to keep my <div v-if="haveStats"> because there can be only one root node.

Reply:
left another comment about this, but can you just change `div` to `template` there?